data/manager.php

```diff
@@ -56,6 +57,13 @@ public static function instance() {
 	}
 
 	public function __construct() {
+		add_action( 'elementor/init', function () {
```

Comment:
The notice shouldn't appear in core (only for 3rd that still uses it)

Reply:
Handled https://github.com/elementor/elementor/pull/14205/commits/c5a31de9983c6d4014dce754b47d3c0d4fe8cf11